src/main/scala/ai/privado/rulevalidator/YamlFileValidator.scala

```diff
@@ -147,12 +150,24 @@ object YamlFileValidator {
       case CatLevelOne.THREATS     => Right(THREATS)
       case CatLevelOne.COLLECTIONS => Right(COLLECTIONS)
       case CatLevelOne.SINKS       => Right(SINKS)
+      case _ =>
+        matchSchemaConfigFile(ruleFile, ruleJsonTree, callerCommand)
+    }
+  }
+
+  def matchSchemaConfigFile(
+    ruleFile: File,
+    ruleJsonTree: JsonNode,
+    callerCommand: String = ""
+  ): Either[Unit, String] = {
+    val configTypeKey =
+      if (ruleJsonTree.fieldNames().hasNext) ruleJsonTree.fieldNames().next() else CatLevelOne.UNKNOWN.name
+    ConfigRuleType.withNameDefaultHandler(configTypeKey) match {
+      case ConfigRuleType.SEMANTICS  => Right(SEMANTICS)
```

Comment:
@abhstabs Instead of creating a new DS, add `semantics` and `exclusions` in `CatLevelOne` similar to how we have for `Threats, Collections, sinks` etc

Reply:
Can we do this without breaking existing functionality? @HiteshMah-Jan, can you confirm?

Reply:
existing functionality should work fine. Although I believe CatLevelOne specifically contains all primary key names for rules, not sure if we should add configs in those.